content/_index.md

```diff
@@ -64,9 +72,5 @@ in order to achieve these goals.
 [wiki-img]: https://img.shields.io/badge/docs-Wiki-blue.svg
 [wiki]: https://github.com/sqlitebrowser/sqlitebrowser/wiki
 
-# Windows Code-Signing Policy
```

Comment:
Nah, we need to leave this bit in.  It's a requirement for the Signpath code signing for Windows that we're working through.

Reply:
OK